src/renderer/widgets/CreateWallet/ui/MultisigWallet/FlexibleMultisigWallet.tsx

```diff
@@ -134,20 +106,9 @@ export const MultiChainMultisigWallet = ({ isOpen, onClose, onComplete, onBack }
           />
 
           {/* Should be before SelectSignatories to avoid hidden nova wallet icon */}
-          {activeStep === Step.CONFIRMATION && (
-            <ConfirmSignatories wallets={signatoryWallets} contacts={signatoryContacts} />
-          )}
-
-          <SelectSignatories
-            isActive={activeStep === Step.INIT}
-            wallets={wallets}
-            accounts={wallets.map((wallet) => wallet.accounts).flat()}
-            contacts={contacts}
-            onSelect={(wallets, contacts) => {
-              setSignatoryWallets(wallets);
-              setSignatoryContacts(contacts);
-            }}
-          />
+          {/* {activeStep === Step.CONFIRMATION && <ConfirmationStep />} */}
```

Comment:
commented code. Check comment line above, is is actual if this code commented?

Reply:
This whole file isn't used right now. I kept it here because it'll be used soon. But if you prefer I can totally remove it either.

Reply:
I removed the whole file

Reply:
oh thanks, somehow I forgot.